packages/service-library/src/servicelib/file_constants.py

```diff
@@ -0,0 +1,5 @@
+from typing import Final
```

Comment:
here I am a bit confused by the module name.
we are here in servicelib. file_constants.py. and these are only related to the simcore_agent files right?
maybe rename that to simcore_agent_constants.py at least?